src/postgres.js

```diff
@@ -26,10 +26,67 @@ squel.flavours['postgres'] = function(_squel) {
     }
   }
 
+  // WITH
+  cls.WithBlock = class extends cls.Block {
```

Comment:
Please shift this class into the part of the file which contains all the block definitions (it's higher up!).